.github/workflows/sycl-post-commit.yml

```diff
@@ -52,6 +52,8 @@ jobs:
             runner: '["Linux", "arc"]'
             extra_lit_opts: --param matrix-xmx8=True
             reset_intel_gpu: true
+          - name: Intel Battlemage Graphics with Level Zero
```

Comment:
I think the default at line 85 is weird and I'd rather see an explicit `target_devices` here.